src/domains/point_set.jl

```diff
@@ -5,10 +5,12 @@
 """
     PointSet(coords)
 
-A set of points with coordinate matrix `coords`. The number of rows
-of the matrix is the dimensionality of the domain whereas the number
-of columns is the number of points in the set. Alternatively, `coords`
-can be a vector of tuples (i.e. points).
+A set of points with coordinates vector `coords`. The vector
+contains many coordinates (number of points), where each coordinate
+is implemented by a static array of length same as the dimensionality.
+To build the PointSet a Matrix can be use. The matrix is the dimensionality 
+of the domain whereas the number of columns is the number of points in the set
+or a Vector of tuples that will be converted to a Vector of static array.
```

Comment:
The docstring can be more concise:

>A set of points with coordinates `coords`. Each point is represented by a static vector
>or tuple. Alternatively, `coords` can be a matrix where the number of rows equals the
> number of dimensions.

Reply:
But a point is indeed represented by a static vector, not a tuple. It can be initialised by a tuple.

Reply:
I like more:

> 
> A set of points with coordinates `coords`. Each point is internally represented by a static
> array. To build the PointSet a vector of tuples or a matrix can be use. 
> The matrix must be of rank 2, where the number of columns corresponds to
> the number of points and the number of rows is the dimensionality of each point

Reply:
Your suggestion is too verbose with a lot internal detail. A docstring is intended for end users who just need to know that they should provide a list of vectors or tuples. Can you please use the docstring I suggested above? It can even be made clearer if you replace `set of points` by `list of points`. Thanks.

Reply:
I just was trying to keep as closer as possible to the doc before. But I can change it to what you propose.